azext_iot/_params.py

```diff
@@ -420,9 +423,15 @@ def load_arguments(self, _):
     with self.argument_context('iotcentral device-twin show') as context:
```

Comment:
Just curious is there a removal of the deprecated 'iotcentral' in favor of 'iot central' in the works?

Reply:
I asked about this, it's still planned but no work has started on it yet.  The IoT Central team is focusing on stabilizing the ignite features first